lib/y2snapper_common.pm

```diff
@@ -104,30 +104,16 @@ sub y2snapper_clean_and_quit {
     # Ensure yast2-snapper is not busy anymore
     wait_still_screen;
     # C'l'ose the snapper module
-    send_key "alt-l";
+    wait_screen_change { send_key "alt-l"; };
 
-    my $ret;
     if ($ncurses) {
-        $ret = wait_serial("yast2-snapper-status-0", 30);
+        wait_serial("yast2-snapper-status-0", 180);
```

Comment:
Good catch. I wonder how this code worked at all, as in yast2_snapper we were not echoing exit code at all, I'll fix this